docker/ml-testing-toolkit/spec_files/api_definitions/fspiop_1.1/api_spec.yaml

```diff
@@ -1198,6 +1198,27 @@ paths:
       parameters:
         #Headers
         - $ref: "#/components/parameters/Accept"
+        ## Adding the following query parameters for testing as TTK is enforcing query params validation which should be fixed in TTK
```

Comment:
Why were we missing these?

I.e. is this missing from the FSPIOP swagger?

Reply:
Yes these query parameters are missing from FSPIOP spec. I am not sure if that's a bug or intentional. But TTK is not accepting the query params without having them in spec file. So I added them for testing with a comment.

Reply:
https://github.com/mojaloop/mojaloop-specification/blob/master/fspiop-api/documents/v1.1-document-set/fspiop-v1.1-openapi3.yaml#L1159 <-- interesting that its missing.